clap-utils/src/compute_unit_price.rs

```diff
@@ -0,0 +1,15 @@
+use {crate::ArgConstant, clap::Arg};
+
+pub const COMPUTE_UNIT_PRICE_ARG: ArgConstant<'static> = ArgConstant {
+    name: "compute_unit_price",
+    long: "--with-compute-unit-price",
+    help: "Set the price in micro-lamports of each transaction compute unit",
+};
+
+pub fn compute_unit_price_arg<'a, 'b>() -> Arg<'a, 'b> {
+    Arg::with_name(COMPUTE_UNIT_PRICE_ARG.name)
+        .long(COMPUTE_UNIT_PRICE_ARG.long)
+        .takes_value(true)
+        .value_name("MICRO-LAMPORTS")
```

Comment:
I thought one of the outcomes of https://github.com/solana-labs/solana/pull/26093 was the decision not to expose the term `micro-lamport` to users?
I'm not totally sure what to put here instead; perhaps we need that terminology entry after all...?

Reply:
Good call! How about just repalce with plain "compute-unit-price", updated -h text as. well

Reply:
Nice, that works!